This allows us to access some function from wchar.h that other platforms
can get from stdio.h. 
Potentially add an ifdef Solaris statement to try and send upstream.

diff --git a/intl/hyphenation/hnjalloc.h b/intl/hyphenation/hnjalloc.h
--- a/intl/hyphenation/hnjalloc.h
+++ b/intl/hyphenation/hnjalloc.h
@@ -24,6 +24,7 @@
  */
 
 #include <stdio.h> /* ensure stdio.h is loaded before our macros */
+#include <wchar.h>
 
 #undef FILE 
 #define FILE hnjFile
